### DISCUSSION TITLE
Please, add an icon or another tittle colour for custom maps

### DISCUSSION THREAD

**Discussion by Ariesun24** [ihhub/fheroes2#8737] [Ideas]
In the game when I click on the option to play a single map I can see if it is for the original game or the expansion. If I add maps I can't distinguish between them unless I have a list on paper or in a text file. An icon to distinguish the edited maps from the original maps would be nice.

**Comment by ihhub** [comment]
@Ariesun24 , isn't it this?

![image](https://github.com/ihhub/fheroes2/assets/19829520/27f631e0-70d0-4cba-8cde-e88706911046)

**Reply by Ariesun24** [reply]
> @Ariesun24 , isn't it this?
> 
> ![image](https://private-user-images.githubusercontent.com/19829520/333804971-27f631e0-70d0-4cba-8cde-e88706911046.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY2NDY1MjMsIm5iZiI6MTcxNjY0NjIyMywicGF0aCI6Ii8xOTgyOTUyMC8zMzM4MDQ5NzEtMjdmNjMxZTAtNzBkMC00Y2JhLThjZGUtZTg4NzA2OTExMDQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI1VDE0MTAyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkxZmJlODUwNDE2YzQ5OTI2OWQ4NzczODI0MGRjZjQ3NTJkMmI3NDM1YWVkOWFiODVkY2IwYmRlMTE2NDE4ZjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eOuS5s1KlBYszan7tNfb7Syb6IC3YK20juRfyoJJb1s)

No, it is not. I mean identifying the original maps (vanilla and expansion) from those created later by players. By the way, I've never seen the sword icon before.

**Reply by ihhub** [reply]
There no such a thing as separation of original maps and created by players. There are maps created by Succession Wars Editor and Price of Loyalty Editor.

**Reply by Ariesun24** [reply]
> No existe tal cosa como la separación de los mapas originales y creados por los jugadores. Hay mapas creados por el editor de Succession Wars y el editor de Price of Loyalty.

And couldn't a different colour be implemented for unofficial maps?

**Reply by oleg-derevenetz** [reply]
> And couldn't a different colour be implemented for unofficial maps?

How do we know that the particular map is "unofficial" and what is "official" and "unofficial" in general? For instance, the official GoG distribution includes many maps created by fans. Are they "official" or "unofficial"?
